src/console_node/nodes.go

```diff
@@ -118,18 +129,19 @@ func doGetNewNodes() {
 		if numRvr > 0 || numMtn > 0 {
 			// NOTE: this should be the ONLY place where the maps of
 			//  current nodes is updated!!!
-			log.Printf("Acquiring new nodes: %d, %d", numMtn, numRvr)
+			// NOTE: paradise nodes are included in mountain count
 			newNodes := acquireNewNodes(numMtn, numRvr, podLocData)
 			// process the new nodes
 			for i, node := range newNodes {
-				log.Printf("  Processing node: %s", node.String())
+				//log.Printf("  Processing node: %s", node.String())
```

Comment:
Same question -- should we just remove this? Or put it behind some conditional, if you're keeping it here for future debugging?

Reply:
Ibid.